### PR TITLE
Fix dead links to Zowe CLI Plug-ins

### LINKED ISSUE #18788: Che4z: Dead links for Zowe CLI plug-ins
These links are dead. Content preventively commented out.

 The following Zowe CLI plug-ins are included in the Che4z basic stack:

    CA Endevor SCM plug-in for Zowe CLI

    CA File Master Plus plug-in for Zowe CLI

    IBM Db2 plug-in for Zowe CLI

    IBM CICS plug-in for Zowe CLI

URL        `https://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-mainframe-software/devops/ca-brightside/3-0/ca-brightside-command-line-interface-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ca-endevor-scm.html'
Parent URL http://localhost:4000/che-7/extensions/eclipse-che4z/, line 933, col 4
Real URL   https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/ca-brightside-command-line-interface-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ca-endevor-scm.html
Result     Error: 404 Not Found

URL        `https://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-mainframe-software/devops/ca-brightside/3-0/ca-brightside-command-line-interface-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ca-file-master-plus.html'
Parent URL http://localhost:4000/che-7/extensions/eclipse-che4z/, line 936, col 4
Real URL   https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/ca-brightside-command-line-interface-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ca-file-master-plus.html
Result     Error: 404 Not Found

URL        `https://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-mainframe-software/devops/ca-brightside/3-0/ca-brightside-command-line-interface-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ibm-db2-database.html'
Parent URL http://localhost:4000/che-7/extensions/eclipse-che4z/, line 939, col 4
Real URL   https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/ca-brightside-command-line-interface-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ibm-db2-database.html
Result     Error: 404 Not Found
 1 thread active,    34 links queued, 1410 links in 1490 URLs checked, runtime 6 minutes, 26 seconds

URL        `https://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-mainframe-software/devops/ca-brightside/3-0/ca-brightside-command-line-interface-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ibm-cics.html'
Parent URL http://localhost:4000/che-7/extensions/eclipse-che4z/, line 942, col 4
Real URL   https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/ca-brightside-command-line-interface-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ibm-cics.html
Result     Error: 404 Not Found

### PR DESCRIPTION
**Pull request by zeibura**
> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?
Fixes links to Zowe CLI Plug-ins

### What issues does this PR fix or reference?

fixes  https://github.com/eclipse/che/issues/18788

### PR Checklist

As the author of this Pull Request I made sure that:

- [x] `vale` has been run successfully against the PR branch
- [ ] Link checker has been run successfully against the PR branch
- [x] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [x] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [x] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [x] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

**Comment by zeibura** [comment]
I'm not sure why the "link checker" bot is failing, it seems to have a problem with a link "https://websocketstest.com/" which is nowhere in this file.

**Comment by themr0c** [comment]
Yes, this error started today, see  https://github.com/eclipse/che/issues/18947

**Comment by themr0c** [comment]
@zeibura If you want that the link appears again, you need to uncomment the lines.

**Comment by zeibura** [comment]
Which link? Uncomment which lines?

**Review by benoitf — APPROVED** [review]

**Comment by benoitf** [comment]
it will require https://github.com/eclipse/che-docs/pull/1832 to be merged and rebased to have linkchecker PR check passing

File: modules/extensions/partials/assembly_eclipse-che4z.adoc [hunk 1]
```diff
@@ -125,10 +125,10 @@ https://join.slack.com/t/che4z/shared_invite/enQtNzk0MzA4NDMzOTIwLWIzMjEwMjJlOGM
...
-// * https://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-mainframe-software/devops/ca-brightside/3-0/ca-brightside-command-line-interface-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ibm-cics.html[IBM CICS plug-in for Zowe CLI]
+// * https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/zowe-cli/available-cli-plug-ins/ca-endevor-scm-plug-in-for-zowe-cli.html[CA Endevor SCM plug-in for Zowe CLI]
+// * https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/zowe-cli/available-cli-plug-ins/ca-file-master-plus-plug-in-for-zowe-cli.html[CA File Master Plus plug-in for Zowe CLI]
+// * https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/zowe-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ibm-db2-database.html[IBM Db2 plug-in for Zowe CLI]
+// * https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/zowe-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ibm-cics.html[IBM CICS plug-in for Zowe CLI]
```
Comment (themr0c) [inline modules/extensions/partials/assembly_eclipse-che4z.adoc]: The links are commented out and don't appear in the docs. If they need to appear again, uncomment them, like this:

```suggestion
* https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/zowe-cli/available-cli-plug-ins/ca-endevor-scm-plug-in-for-zowe-cli.html[CA Endevor SCM plug-in for Zowe CLI]
* https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/zowe-cli/available-cli-plug-ins/ca-file-master-plus-plug-in-for-zowe-cli.html[CA File Master Plus plug-in for Zowe CLI]
* https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/zowe-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ibm-db2-database.html[IBM Db2 plug-in for Zowe CLI]
* https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/zowe-cli/available-cli-plug-ins/ca-brightside-plug-in-for-ibm-cics.html[IBM CICS plug-in for Zowe CLI]
```

**Review by themr0c — CHANGES_REQUESTED** [review]

**Review by themr0c — APPROVED** [review]
Let's move forward

**Comment by themr0c** [comment]
One last rebase and we can merge this. Thank you, @zeibura.

**Pull request merged**